src/tests/dejagnu/t_inetd.c

```diff
@@ -82,7 +82,8 @@ main(argc, argv)
 
     progname = argv[0];
 
-    if(argc <= 3) usage();
+    if (argc < 3)
```

Comment:
Not sure I understand this change.  Line 139 references `argv[3]` unconditionally, so I think the original is right?

Reply:
It turns out this isn't right.  t_inetd.c wants the program name specified twice--once for its path and again for argv[0].  I misunderstood the code; what we actually need is an adjustment to start_in_inetd() in k5test.py.